Userland/Libraries/LibCore/StandardPaths.cpp

```diff
@@ -229,6 +242,9 @@ ErrorOr<Vector<String>> StandardPaths::font_directories()
         // FIXME: We should be using the ASystemFontIterator NDK API here.
         // There is no guarantee that this will continue to exist on future versions of Android.
         "/system/fonts"_string,
+#    elif defined(AK_OS_WINDOWS)
+        "C:\\Windows\\Fonts"_string,
+        TRY(String::formatted(R"(C:\Users\{}\AppData\Local\Microsoft\Windows\Fonts)"sv, getenv("USERNAME"))),
```

Comment:
```suggestion
        TRY(String::formatted(R"({}\AppData\Local\Microsoft\Windows\Fonts)"sv, home_directory())),
```

or, probably even better,
```suggestion
        TRY(String::formatted(R"({}\Microsoft\Windows\Fonts)"sv, getenv("LOCALAPPDATA"))),
```